build.gradle

```diff
@@ -22,6 +23,31 @@ spotless {
     }
 }
 
+cyclonedxBom {
+    // includeConfigs is the list of configuration names to include when generating the BOM (leave empty to include every configuration), regex is supported
+    includeConfigs = ["runtimeClasspath"]
+    // skipConfigs is a list of configuration names to exclude when generating the BOM, regex is supported
+    skipConfigs = ["testCompileClasspath", "testRuntimeClass", "testRuntimeOnly"]
+    // skipProjects is a list of project names to exclude when generating the BOM
+    skipProjects = ["integration-test", "demo", "smoketest"]
+    // Specified the type of project being built. Defaults to 'library'
+    projectType = "library"
+    // Specified the version of the CycloneDX specification to use. Defaults to '1.5'
+    schemaVersion = "1.5"
+    // Boms destination directory. Defaults to 'build/reports'
+    destination = file("artifacts/sboms")
+    // The file name for the generated BOMs (before the file format suffix). Defaults to 'bom'
+    outputName = "sbom_without_team_name"
+    // The file format generated, can be xml, json or all for generating both. Defaults to 'all'
+    outputFormat = "json"
+    // Exclude BOM Serial Number. Defaults to 'true'
+    includeBomSerialNumber = false
+    // Exclude License Text. Defaults to 'true'
+    includeLicenseText = true
+    // Override component version. Defaults to the project version
+    componentVersion = "2.0.0"
```

Comment:
This variable is not necessary. It will pick up the project version and that's totally fine.